packages/core/src/Table/TableCellHead/Action.js

```diff
@@ -0,0 +1,29 @@
+import { colors } from '@dhis2/ui-constants'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export const Action = ({ onClick, children }) => (
+    <span onClick={onClick}>
```

Comment:
Using a `button` instead of a `span` would make `Action` keyboard accessible - users would be able to use the enter key to trigger `onClick`.

Reply:
Diff showing style changes needed:

```diff
diff --git a/packages/core/src/Table/TableCellHead/Action.js b/packages/core/src/Table/TableCellHead/Action.js
index deeec87..a881417 100644
--- a/packages/core/src/Table/TableCellHead/Action.js
+++ b/packages/core/src/Table/TableCellHead/Action.js
@@ -3,10 +3,12 @@ import PropTypes from 'prop-types'
 import React from 'react'
 
 export const Action = ({ onClick, children }) => (
-    <span onClick={onClick}>
+    <button onClick={onClick}>
         {children}
         <style jsx>{`
-            span {
+            button {
+                border: none;
+                padding: 0;
                 background: transparent;
                 width: 24px;
                 height: 36px;
@@ -15,12 +17,13 @@ export const Action = ({ onClick, children }) => (
                 justify-content: center;
                 flex-shrink: 0;
             }
-            span:hover {
+            button:hover, button:focus {
+                outline: none;
                 background: ${colors.grey400};
                 cursor: pointer;
             }
         `}</style>
-    </span>
+    </button>
 )
 
 Action.propTypes = {
```